### ISSUE TITLE
Replication Mate

### ISSUE THREAD

**Issue by oeresundsgruppen** [SolaceProducts/terraform-provider-solacebroker#67]
Hi, can replication mate be set using SEMP or Terraform provider ?

**Comment by bczoma** [comment]
Hi @oeresundsgruppen , this is not supported in SEMP yet and hence in the provider which builds on SEMP. This initial Beta and soon to be GA release currently supports configuration of Message VPN level services. See this announcement for additional details: https://solace.community/discussion/3292/solace-beta-opportunity-solace-terraform-software-provider-declarative-semp

**Comment by oeresundsgruppen** [comment]
ok, it needs to be in SEMP first I guess, seems it can be done via environment config vars for the time being...

**Comment by oeresundsgruppen** [comment]
Actually its just redundancy, which is supported via env vars, not replication mate links - strange that this is left to be 100% manual set up...

**Comment by bczoma** [comment]
Indeed... I suggest expressing interest through [Solace Support](https://solace.com/support/) or at [Solace community](https://solace.community/post/question)
